docs/develop/terrain/initial-setup.md

```diff
@@ -56,10 +56,16 @@ Make sure you are using the latest version of Terrain. Previous versions may cau
 terrain --version
 ```
 
-If you installed Terrain from a previous, make sure to update it: 
+Make sure to check if you have another Terrain package installed globally.
 
 ```sh
-npm uninstall -g @iboss/terrain
+npm list -g
+```
+
+And update the package if necessary.
+
+```sh
+npm uninstall -g @other/terrain
```

Comment:
For clarity and to remove the redundant language of "make sure," try something like the following: 

Check if you have another Terrain package installed globally:

```sh
npm list -g
```

Update the package if necessary:

```sh
npm uninstall -g @<outdated-terrain-package>

Reply:
@evanorti rearranged some things.  Let me know what you think!

Reply:
LGTM